QUEST_JOBSTEP.tsv

```diff
@@ -753,42 +753,42 @@ QUEST_JOBSTEP_20150317_000752	The disciples came to Lydia Schaffen. Afterwards,
+QUEST_JOBSTEP_20150317_000760	$Compared to Ruklys, who likely died early on during the Civil War, or Flurry, who rarely leaves the tower she built, Lydia Schaffen definitely went on a greater number of adventures around the world, and the stories from her adventures include characters that she was well acquainted with. {nl}From them, even an adversary's fable could be a part of her stories. {nl}
+QUEST_JOBSTEP_20150317_000761	$It is not known how true the story really is. Perhaps it was just as absurd in the past as it would definitely be absurd in the present. Still, this kind of story brings a perspective in life, and the fact that it is still told today means the story agrees with the nature of Lydia Schaffen.
+QUEST_JOBSTEP_20150317_000762	$One day Lydia Schaffen sneaked out of a certain village alone and walked around at night.{nl} Lydia Schaffen saw a traveler sitting on a woodland road. {nl} Out of curiosity, she decided to ask him.{nl} 'Why are you sitting here in the forest in the middle of the night?'{nl}The traveler was hesitant to answer up to this point, but after being asked multiple times he finally opened his mouth. {nl}The reason was apparently this. {nl}
+QUEST_JOBSTEP_20150317_000763	$The traveler had met a passing demon, named Rumpelstiltskin, in the forest several days ago. {nl}He was invited to play a game with the demon. {nl}
+QUEST_JOBSTEP_20150317_000764	$The reward for winning the game was significant, but the game with the demon was not easy. In the end, the traveler lost to Rumpelstiltskin. {nl}And when traveler reached the part where he had to pay for losing the game, he took a short breath, was told to keep waiting. {nl}After hearing the conditions, Lydia Schaffen started to search for the troublemaker, Rumpelstiltskin.
+QUEST_JOBSTEP_20150317_000765	$Lydia quickly liberated the traveler of his conditions, and in exchange, sent a challenge request to Rumpelstiltskin. {nl}Rumpelstiltskin was well acquainted with Lydia Schaffen's reputation, and the fact that she requested a game with him excited him into accepting her request. {nl}
+QUEST_JOBSTEP_20150317_000766	$Although Rumpelstiltskin is not well known by people today, he played often and was famous for gifting those who lost in his games in the past. {nl} But, his games have always ended with little variation. {nl} Lydia Schaffen proposed, 'I ask for a game related to archery, but what sort of thing should we do?'{nl}
+QUEST_JOBSTEP_20150317_000767	$'Of course, a game to see who can shoot better with a bow would not be that interesting. Therefore, let's play a game where you fire an arrow with your bow at the target, and then, I will try to catch and ride on the arrow before the arrow reaches the target,' said Rumpelstiltskin. {nl}Lydia replied back, questioning the proposal, 'Ride on the arrow?'{nl}'That is correct,' averred Rumpelstiltskin.{nl}
```

Comment:
The period is after Rumpelstiltskin; Dialogues in English where the speaker is noted afterwards have commas to "end" them.